.github/workflows/javascript.yaml

```diff
@@ -0,0 +1,21 @@
+name: JavaScript
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    paths:
+      - "javascript/**"
+      - .github/workflows/rust.yaml
+jobs:
+  coverage:
+    name: Test
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run tests
+        run: make test-javascript
```

Comment:
looks like an exit status is being ignored somewhere because https://github.com/swift-nav/libsbp/pull/905/checks?check_run_id=2124840858#step:3:479 doesnt look very successful.

may be related to https://github.com/swift-nav/libsbp/issues/927